dev_guide/deployments.adoc

```diff
@@ -273,6 +273,14 @@ image stream changes and the new tag value differs from the current image
 specified in the deployment configuration's `helloworld` container, a new
 deployment is created using the new tag value for the `helloworld` container.
 
+The following table explains the possible combinations between ConfigChange
+and ImageChange triggers.
+
+  Trigger                  | true                                      | false
+  -----------------------  | ----------------------------------------- | ------------------------------------------------------
+  ConfigChange             | Deployments are started automatically     | Deployments need to be started manually
+  ImageChange (automatic)  | Images are always resolved automatically  | Only the initial image will be resolved in the template
```

Comment:
I don't need the ICT in this case.  It points to an imagestream that is never
updated.  I'm not doing any builds since I know for certain that the images
will already be on the node.  I also have the `IfNotPresent` pull policy
because of that.  As I said, not a common setup.

The thing is: everything works fine if I have only the CC trigger.  It seems
that the current situation is: if I have an ICT I am forced to populate the IS
it points to, even if the ICT is not used and all I need is the CCT.  From what
I understand, this "forced dependency" that the ICT introduces on the CCT is a
side-effect of our race-prevention code.  I still think that's surprising.